esp/templates/survey/questions/favorite_class.html

```diff
@@ -2,11 +2,12 @@
 {% load survey %}
 <table class="fullwidth">
 <tr>
-    <td width="30%"><b>{{ name }}</b></td>
-    <td width="70%">
-        {% templatetag openblock %} for l in lst {% templatetag closeblock %}
+    <td>
+    <div style = "text-align: left; font-weight: bold;"> {{ name }} </div>
+    <div style="text-align: left; padding-left:2em; text-indent:-2em;">{% templatetag openblock %} for l in lst {% templatetag closeblock %}
         <input type="checkbox" name="question_{{ id }}" value="{% templatetag openvariable %} l.id {% templatetag closevariable %}" />{% templatetag openvariable %} l.emailcode {% templatetag closevariable %}: {% templatetag openvariable %} l.title {% templatetag closevariable %}<br />
         {% templatetag openblock %} endfor {% templatetag closeblock %}
+    </div>
```

Comment:
I think you want each checkbox/answer pair in its own div like with the other multiple choice question types below, otherwise you end up with something like this:
![image](https://user-images.githubusercontent.com/7232514/87878897-ca12a980-c9ac-11ea-8edd-6de89e6fe59d.png)

Reply:
If you're having trouble debugging this, you can register dummy classes through teacher registration